hack/extract-cm.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+
+# Copyright 2015 The Kubernetes Authors.
```

Comment:
nit:
```suggestion
# Copyright 2022 The Kubernetes Authors.
```

Reply:
lol, why did I have to copy-paste from an old file :-)
thanks, fixed